android/app/src/main/java/org/chrome/device/ml/tokenization/CharChecker.java

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2020. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+package org.chrome.device.ml.tokenization;
+
+/**
+ * To check whether a char is whitespace/control/punctuation.
+ */
+final class CharChecker {
+
+    /**
+     * To judge whether it's an empty or unknown character.
```

Comment:
Returns whether ... 

here and below

Reply:
this is the third party library. I'll move it to a different directory to be more clear